docs/data/joy/components/tooltip/tooltip.md

```diff
@@ -61,7 +61,7 @@ You can use the `arrow` prop to give your tooltip an arrow indicating which elem
 
 ## Common examples
 
-### Github tooltip
+### GitHub tooltip
```

Comment:
How did you manage to find this typo :D

Reply:
I have seen people get it wrong all the times, I did a case sensitive search on the codebase.